src/nunit-gui/Program.cs

```diff
@@ -43,7 +43,11 @@ static void Main(string[] args)
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
+            Run(args);
+        }
 
+        public static void Run(string[] args)
```

Comment:
Would you add a comment about why we have this separate entry?

Reply:
More than happy to. :)

Reply:
Done. How's that?